### PR TITLE
Add a bunch of UX tweaks to v2

### PR DESCRIPTION
**Pull request by schlosser**
Still working! ⚠️ 

See details here: https://docs.google.com/document/d/1Kl1EGt3D5SalVP7ugjv_GRB3H1yueD095XkzlyuKi1I/edit#

### CODE REVIEW

File: src/components/Charts/ModelChart.js [hunk 1]
```diff
@@ -350,38 +362,17 @@ const ModelChart = ({
...
                 title="Currently we aggregate data over 4 day intervals to smooth out inconsistencies in the source data. We’re working on improving this now."
                 placement="bottom"
               >
-                <span>
-                  <strong>
-                    Last updated {lastUpdatedDate.toLocaleDateString()}
-                  </strong>
-                  .{' '}
-                </span>
+                <span>Last updated {getDateUpdated()}</span>
```
Comment (igorkofman) [inline src/components/Charts/ModelChart.js:372]: What is the purpose of this?

File: src/components/Charts/ModelChart.style.js [hunk 2]
```diff
@@ -1,36 +1,50 @@
...
-  color: rgba(0, 0, 0, 0.7);
+export const DisclaimerWrapper = styled.div`
+  display: flex;
+  margin: 0 -0.75rem;
 
-  @media (min-width: 900px) {
-    text-align: center;
-    margin: 0 0 2rem 0;
+  @media (max-width: 600px) {
+    flex-direction: column;
```
Comment (igorkofman) [inline src/components/Charts/ModelChart.style.js:18]: https://www.dropbox.com/s/x9e580su829dli0/Screenshot%202020-04-18%2021.53.08.png?dl=0

Padding seems off

File: src/screens/Endorsements/Endorsements.js [hunk 3]
```diff
@@ -46,7 +22,7 @@ const Endorsements = () => {
             </Grid>
           ))}
         </EndorsersWrapper>
-        <ShareBlock />
+        {social && <ShareBlock />}
```
Comment (igorkofman) [inline src/screens/Endorsements/Endorsements.js:25]: is this ever true?

File: src/components/Charts/ModelChart.js (hunk 1 above)
Comment (schlosser) [reply src/components/Charts/ModelChart.js:372]: This is to show a valid date that is within three days of today, so we don't have to update the date updated manually.

File: src/screens/Endorsements/Endorsements.js (hunk 3 above)
Comment (schlosser) [reply src/screens/Endorsements/Endorsements.js:25]: Ah, should be when standalone. Updating!

File: src/components/Charts/ModelChart.js (hunk 1 above)
Comment (igorkofman) [reply src/components/Charts/ModelChart.js:372]: The date is displayed automatically in the current code.

**Pull request merged**